### DISCUSSION TITLE
Welcome to the IOTA SmartContracts X-Team Discussions!

### DISCUSSION THREAD

**Discussion by antonionardella** [iota-community/X-Team_IOTA_SmartContracts#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamadamsiwy** [comment]
Hi... I'm Adam - living in Mannheim South-West Germany. Since October 2020 I observe some very interessting movement in the IOTA Smart Contract space and love to contribute my part to bring this ideas to life.

Topics I can bring forward:

- a community goshimmer node with a small committee of 5-9 distributed wasp nodes
- UX and approachable demos for the WASP Suite
- Motivation of Experimenting with the existing tools (e.g. german Mini Tutorials https://youtu.be/5KLp9j9ES2A / https://youtu.be/APRqB5jrYdU / https://youtu.be/PG9ya8nThXg) - now with the new tutorials a great point to record english mini-tutorials
- Smart Contract Concepts for sharing and resharing physical items via Smart 'Sharing' Contracts for my personal fljota.network project, witch is a big opportunity for a ISCP PoC
- wide architectional, conceptional and usercentered ideas

What I need to learn:
- Using Solo for my demo case
- Deploying a Community Chain with 6-24 Contracts
- Some Rust & Go skills, but it's not my biggest motivation
- WASM

What would be nice:
- Building Community Node Access with Fido2 WebAuth
- Wasp Dashboard with Approachable Interaction and API Demos
- ...
